src/test/ui/generic-associated-types/issue-79422.rs

```diff
@@ -0,0 +1,47 @@
+#![feature(generic_associated_types)]
+#![allow(incomplete_features)]
```

Comment:
Can you remove this? I think it's probably good to have this so we don't forget to remove it later when this is no longer incomplete.

Reply:
Actually, seems like you deleted the wrong line 😂

`#![feature(generic_associated_types)]`